agent/agent.go

```diff
@@ -404,6 +428,21 @@ func (a *Agent) Run() error {
 		logger.WithError(err).Error("error sending keepalive")
 	}
 
+	go func() {
+		// TODO(cyril): create a separate config knob for system info refresh interval?
```

Comment:
Would this be a user-configurable agent config attribute, or a default based on the keepalive value (as it is now)?

Reply:
I'm assuming there is little value for the user to be able to configure it, except maybe in extreme edge cases such as the user "needing" to change/reconfigure network interfaces more frequently than the system information is refreshed?

I would be happy with a default value of our choosing, say `DefaultSystemInfoRefreshInterval = 60`, that wouldn't be exposed in the user config:
- we can change it at will at compile time
- if it really becomes a valid need for users we can then expose it